services/ircd/inspircd/conf.nix

```diff
@@ -386,7 +386,7 @@ in {
       certfile = "${config.security.acme.certs."irc.${tld}".directory}/fullchain.pem";
       keyfile = "${common.certsDir}/irc.${tld}/key.pem";
       dhfile = config.security.dhparams.params.ircd.path;
-      hash="sha256";
+      hash="sha1";
```

Comment:
Why is this needed?

Reply:
i only found the above broken when I put it into prod, ircd needs a testing env